src/Pkg.jl

```diff
@@ -311,12 +311,14 @@ const project = API.project
 """
     Pkg.instantiate(; verbose = false)
 
-If a `Manifest.toml` file exists in the current project, download all
+If a `Manifest.toml` file exists in the active project, download all
 the packages declared in that manifest.
 Otherwise, resolve a set of feasible packages from the `Project.toml` files
 and install them.
 `verbose = true` prints the build output to `stdout`/`stderr` instead of
 redirecting to the `build.log` file.
+If no `Project.toml` exist in the current active project, create one with all the
+dependencies in the manifest.
```

Comment:
```suggestion
dependencies in the manifest, then install all the packages declared in the manifest.
```
Perhaps this is more clear that you proceed with a normal `instantiate` after the project file is created.